httpserver-rs/settings.md

```diff
@@ -90,4 +95,6 @@ The link definition connecting an actor to a capability provider includes a key-
   ```
   (The option `-w0` omits line breaks within the base64)
 
-- use key `config_json`, with a value that is the raw json. Don't forget to escape quotes.
\ No newline at end of file
+- use key `config_json`, with a value that is the raw json. Don't forget to escape quotes.
+
+- use key `readonly_mode`, with a boolean value to set the mode as true or false.
```

Comment:
I just realized this section is really confusing. It doesn't mention any of the other values besides `config_file` and `config_b64` which are used for bulk settings... I would either add the rest of the settings to this section (sorry), or leave this out